resonate/retry_policies/constant.py

```diff
@@ -8,10 +9,15 @@
 @dataclass(frozen=True)
 class Constant:
     delay: float = 1
-    max_retries: int = -1
+    max_retries: int = sys.maxsize
 
     def next(self, attempt: int) -> float | None:
         assert attempt >= 0, "attempt must be greater than or equal to 0"
```

Comment:
Let's then assert `attempt > 0`

Reply:
I tweaked 0 to indicate the first attempt

Reply:
Oh ok! got it